server/controllers/index.js

```diff
@@ -208,8 +123,7 @@ export const explore = async(ctx, next) => {
     topPromo,
     second3Promos,
     next8Promos,
-    collectorsPromo,
-    latestTweets
```

Comment:
I believe this was the unnecessary offending synchronous request for latest tweets that has caused page load to increase.